plugin.yml

```diff
@@ -1,8 +1,8 @@
 ---
 name: MagicWE2
 main: xenialdan\MagicWE2\Loader
-version: 6.0.2
-api: [4.0.0+dev]
+version: 7.0.0
+api: [4.0.0,3.0.0]
```

Comment:
As @SalmonDE stated, changes in 4.0.0 are not backwards compatible with 3.0.0.

i.e. `https://github.com/pmmp/PocketMine-MP/blob/master/src/pocketmine/lang/Language.php` > `https://github.com/pmmp/PocketMine-MP/blob/mcpe-1.8/src/pocketmine/lang/BaseLang.php`

Reply:
Sadly i must have it enabled because my 1.8 pmmp source uses the 3.0.0 api

Reply:
rip